lib/sidekiq/backtrace_cleaner.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Sidekiq
+  # @api private
+  class BacktraceCleaner
```

Comment:
Can we alias `clean` to `call` and use a no-op Proc by default so we don't need to create any classes for this?

Reply:
But rails wants `clean` - https://api.rubyonrails.org/classes/ActiveSupport/BacktraceCleaner.html#method-i-clean
Or where did you mean to add an alias?

Reply:
I mean monkeypatch AS::BacktraceCleaner to add an alias.

Reply:
Added a monkey patch. This monkey patch can be even upstreamed, I think.